Makefile

```diff
@@ -433,8 +443,71 @@ vendor/check: vendor/fix
 .PHONY: vendor/fix
 vendor/fix:
 	go mod tidy
-	go mod vendor
+	go mod vendo
+
+.PHONY: manifest/release
```

Comment:
hey guys, I think this needs a rebase, the manifest/release was replaced with manifest/prodsec, without it both the prepare-release pipeline and the manifest check on this PR will fail.